### PR TITLE
add size test to ci

### PR DESCRIPTION
**Pull request by lucylq**

### CODE REVIEW

File: .github/workflows/pull.yml [hunk 1]
```diff
@@ -239,6 +239,36 @@ jobs:
...
+        CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+        conda activate "${CONDA_ENV}"
+
+        # build module for executorch.extension.pybindings.portable_lib
+        bash test/build_size_test.sh
+        strip cmake-out/test/size_test
+        output=$(ls -la cmake-out/test/size_test)
+        arr=($output)
+        size=${arr[4]}
+        threshold="48120"
```
Comment (larryliu0820) [inline .github/workflows/pull.yml:264]: Is this the current size?

**Review by larryliu0820 — APPROVED** [review]
Looks good but probably want to fix CI job failure

File: .github/workflows/pull.yml (hunk 1 above)
Comment (lucylq) [reply .github/workflows/pull.yml:264]: For my local on devserver, yes. On CI it is 51768

Comment (lucylq) [reply .github/workflows/pull.yml]: ah, seems the difference is gcc vs clang

**Pull request closed without merging**